react/Badge/Readme.md

```diff
@@ -1,6 +1,6 @@
 `<Badge />` component wraps the element you want to apply a badge to. It is based on the [Badge component from Material-UI](https://v4.mui.com/api/badge/).
 
-### Basic Badges
+### Basic badges
```

Comment:
this file is screenshoted by Argos. It's better to update a file not in the Argos pipe to avoid false positive. It still triggers Argos.

Reply:
Fixed